astropy/coordinates/representation/geodetic.py

```diff
@@ -45,6 +46,7 @@ class BaseGeodeticRepresentation(BaseRepresentation):
     to quantities holding correct values (with units of length and dimensionless,
     respectively), or alternatively an ``_ellipsoid`` attribute to the relevant ERFA
     index (as passed in to `erfa.eform`).
+    Longitudes are east positive and span from 0 to 360 degrees by default.
```

Comment:
Nitpick: might as well leave this out for now.

Instead, I think here is the place to mention how lon, lat, and height are defined relative to the ellipsoid.

Alternatively, you could clarify the definition in `geodetic_base_doc` above (and then *not* use it for `bodycentric`)